warehouse/query-core/src/test/java/datawave/query/planner/rules/NodeTransformVisitorTest.java

```diff
@@ -112,7 +117,24 @@ public void regexPushdownAnyfieldTransformRuleTest() throws Exception {
             // ok
         }
     }
-    
+
+    @Test
+    public void regexSimplifierTransformRuleTest() throws Exception {
+        // @formatter:off
+        String query = "BLA == '.*?.*?x' && " +
+                "BLA =~ 'ab.*.*' && " +
+                "BLA =~ 'a.*.*.*.*?.*?' && " +
+                "BLA =~ '.*?.*?.*bla.*?.*?blabla' && " +
+                "_ANYFIELD_ =~ '.*.*?.*?<bla>'";
+        String expected = "BLA == '.*?.*?x' && " +
```

Comment:
shouldn't this be simplified to `BLA == '.*?x'`?

Reply:
That is an equals node, not a regex node.

Reply:
That is an equals node, not a regex node.

Reply:
ha, right. You got me